src/rtc.c

```diff
@@ -0,0 +1,518 @@
+/**
+ * @file rtc.c
+ * @brief Real-Time Clock Subsystem
+ * @ingroup rtc
+ */
+
+#include "libdragon.h"
+
+/**
+ * @defgroup rtc Real-Time Clock Subsystem
+ * @ingroup libdragon
+ * @brief Joybus real-time clock interface.
+ *
+ * The Joybus real-time clock is an accessory that was only included on
+ * one official cartridge that was only available in Japan. The Joybus RTC
+ * is accessed through the serial interface (SI) similar to EEPROM.
+ * A real RTC uses a battery to power a clock that runs even when the N64
+ * is powered-off and maintains the date, time, and day of the week.
```

Comment:
Maybe I'd move this as the very first paragraph to quickly describe what a RTC is.